.github/workflows/qc.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [ "3.8", "3.9", "3.10" ]
+        python-version: [ "3.8", "3.11" ]
```

Comment:
why not check 10 and 9?

Reply:
comes from the cookiecutter

Reply:
Is this wise? I mean, did Chris agree with this?

Reply:
He recommended it in another project. I have seen Charlie do that too.